### PR TITLE
feat: o-forms toggle on white background support

### PR DESCRIPTION
**Pull request by notlee**
Adds .o-forms-field--white

### CODE REVIEW

File: components/o-forms/main.scss [hunk 1]
```diff
@@ -34,20 +34,28 @@
...
 )) {
 	$elements: map-get($opts, 'elements');
 	$features: map-get($opts, 'features');
+	$themes: map-get($opts, 'themes');
 
 	$disabled: index($features, 'disabled');
 	$inline: index($features, 'inline');
-	$inverse: index($features, 'inverse');
+	// For use on a dark background.
+	$inverse-theme: index($features, 'inverse') or index($themes, 'inverse');
```
Comment (chee) [inline components/o-forms/main.scss:56]: nice:)

File: components/o-forms/src/scss/_toggle.scss [hunk 2]
```diff
@@ -34,6 +34,7 @@
 			&:after {
 				@include oIconsContent('cross', $color: _oFormsGet('default-text'), $size: null);
 				background-color: oColorsByUsecase(page, background);
+				background-color: var(--o-forms-toggle-handle-color, #{oColorsByUsecase(page, background)}); //stylelint-disable-line declaration-block-no-duplicate-properties
```
Comment (chee) [inline components/o-forms/src/scss/_toggle.scss:37]: nitpick, but i much prefer `// stylelint-disable-next-line`, trailing comments look yicky to me.

File: components/o-forms/main.scss [hunk 3]
```diff
@@ -125,10 +133,16 @@
...
 
-	@if $inverse {
+	@if $inverse-theme {
 		@include _oFormsInverse($error-summary, $radio-box, $file);
 	}
 
+	@if $white-theme {
+		.o-forms-field--white {
+			--o-forms-toggle-handle-color: #{oColorsByName('white')};
+		}
```
Comment (chee) [inline components/o-forms/main.scss:143]: cool, looking good

**Review by chee — APPROVED** [review]

File: components/o-forms/src/scss/_toggle.scss (hunk 2 above)
Comment (notlee) [reply components/o-forms/src/scss/_toggle.scss:37]:
```suggestion
				//stylelint-disable-next-line declaration-block-no-duplicate-properties
				background-color: var(--o-forms-toggle-handle-color, #{oColorsByUsecase(page, background)});
```

**Comment by notlee** [comment]
Merging. Percy is sad for no reason:
https://github.com/Financial-Times/origami/pull/714#issuecomment-1104531757

**Pull request merged**